source/agora/consensus/SCPEnvelopeStore.d

```diff
@@ -0,0 +1,199 @@
+/*******************************************************************************
+
+    Contains supporting code for restoring the latest SCP envelopes,
+    using SQLite as a backing store.
```

Comment:
```suggestion
    Contains supporting code for storing the latest SCP envelopes,
    using SQLite as a backing store.
```

This code does not deal with restoring anything, just storing the data (being able to load it is implicit). The Nominator is in charge of doing the work of restoring.

Reply:
Yes